github/repos.go

```diff
@@ -293,7 +293,7 @@ func (s *RepositoriesService) Get(ctx context.Context, owner, repo string) (*Rep
 
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
-	acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeSquashPreview}
+	acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeSquashPreview, mediaTypeCodesOfConductPreview}
```

Comment:
This is great. It means GitHub API will send a "code_of_conduct" key in the repository object response.

But you still need to add a ``CodeOfConduct `json:"code_of_conduct,omitempty"` `` to the [`Repository`](https://github.com/google/go-github/blob/9f4675c6fd826772a2fffe74b2a7b6c9822802a0/github/repos.go#L21) struct above.

Otherwise people won't be able to access it.